cime_config/tests.py

```diff
@@ -274,6 +274,7 @@
             "ERS_P480_Ld5.TL319_IcoswISC30E3r5.GMPAS-JRA1p5-DIB-PISMF.mpaso-jra_1958",
             "PEM_P480_Ld5.TL319_IcoswISC30E3r5.GMPAS-JRA1p5-DIB-PISMF.mpaso-jra_1958",
             "SMS_P480_Ld5.TL319_IcoswISC30E3r5.GMPAS-JRA1p5-DIB-PISMF-TMIX.mpaso-jra_1958",
+            "SMS_P480_Ld5.TL319_IcoswISC30E3r5.GMPAS-JRA1p5-DIB-PISMF-DSGR.mpaso-jra_1958",
```

Comment:
Perfect!  This is nearly identical to the test I ran:
```
SMS_P480_Lm1.TL319_IcoswISC30E3r5.GMPAS-JRA1p5-DIB-PISMF-DSGR.chrysalis_intel
```
It just brings in one year of JRA forcing and runs for only 5 days, not a month.  So I'm sure it will work great!